src/main/java/tlylt/haha/TaskList.java

```diff
@@ -0,0 +1,220 @@
+ * hold all the information when program is running.
+ */
+public class TaskList {
+    private final List<Task> database = new ArrayList<>();
+
+    /**
+     * Outputs information about adding of task to database.
+     */
+    String tellAdd() {
+        System.out.println("Got it. I've added this task:");
+        return "Got it. I've added this task:\n";
+    }
+
+    /**
+     * Outputs information about size of database.
+     */
+    String tellSize() {
+        String task = database.size() > 1 ? " tasks" : " task";
+        System.out.println("Now you have " + database.size() + task + " in the list");
+        return "Now you have " + database.size() + task + " in the list\n";
+    }
+
+    /**
+     * Adds task into database.
+     *
+     * @param task Task created by user.
+     */
+    String addToDB(Task task) {
+        String response = "";
+
+        database.add(task);
+        response += tellAdd();
+        System.out.println("  " + database.get(database.size() - 1));
+        response += "  " + database.get(database.size() - 1) + "\n";
+        response += tellSize();
+        return response;
+    }
+
+    /**
+     * Removes task from database.
+     *
+     * @param inputNum String that will be parsed for task number.
+     */
+    String deleteFromDB(String inputNum) {
+        String response = "";
+        try {
+            int num = Parser.taskNumber(inputNum);
+            Task currentTask = database.get(num - 1);
+            System.out.println("Noted. I've removed this task:");
+            System.out.println(currentTask);
+            response += "Noted. I've removed this task:\n";
+            response += currentTask + "\n";
+            database.remove(currentTask);
+            response += tellSize();
+        } catch (HahaTaskNumberNotIntException ex) {
+            System.out.println(ex);
+            return ex.toString();
+        } catch (IndexOutOfBoundsException ex) {
+            System.out.println("OOPS! Wrong number!\nTry specify the right task number");
+            return "OOPS! Wrong number!\nTry specify the right task number\n";
+        }
+        return response;
+    }
+
+    /**
+     * Reads tasks from a file that contains previous usage.
+     *
+     * @param file Content of previously recorded tasks.
+     */
+    void readTasks(List<String> file) {
+        List<Task> tasks = new ArrayList<>();
+        file.forEach(line -> tasks.add(Parser.parseLine(line)));
+        database.addAll(tasks);
+    }
+
+    /**
+     * Updates database to file.
+     */
+    void updateFile() {
+        List<String> str = new ArrayList<>();
+        database.forEach(task -> str.add(task.fileStorageFormat()));
+        try {
+            Files.write(Paths.get(System.getProperty("user.dir"), "Haha_data", "database.txt"),
+                    str, StandardOpenOption.TRUNCATE_EXISTING);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
```

Comment:
I like how your method names in general uses verb, and accurately represents their functionality.